### DISCUSSION TITLE
Is it possible to have AlexaMediaPlayer run without amazon authenticator enabled?

### DISCUSSION THREAD

**Discussion by hefal** [alandtse/alexa_media_player#1302] [Q&A] [answered]
Is it possible to have AlexaMediaPlayer run without amazon authenticator enabled? 

The manual just says that theres no guarantee or well "support" if youre using it without the authenticator enabled on your alexa account..

**Comment by alandtse** [comment, marked as answer]
Yes it's possible. I won't bother to help you if you have login issues though.
